guide_sidebar.py

```diff
@@ -0,0 +1,399 @@
+
+
+def non_complianceTabInfo():
+    # Define the CSS style for the gray rectangle
+    gray_rectangle_style = """
+        background-color: #e6e6e6;
+        padding: 10px;
+        border-radius: 5px;
+        color: black;
+    """
+
+    # Define the HTML content as a string
+    html_content = f"""
+    <div style="{gray_rectangle_style}">
+        <ul>
+            <li>Noncompliance is ipsum dolor sit amet, consectetur adipiscing elit,
+              sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+              Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+              nisi ut aliquip ex ea commodo consequat. Ut enim ad minim veniam, 
+              quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+              commodo consequat.</li>
+            <li>These figures show the ten facilities in this state with the 
+            worst history of environmental compliance based on their number of 
+            noncompliant quarters in the past 3 years (not necessarily consecutive). 
+            Click on a facility name to view its ECHO report.</li>
+        </ul>
+    </div>
+    """
+
+    # Use st.markdown to render the HTML content
+    st.markdown(html_content, unsafe_allow_html=True)
+
+
+def non_complianceTabData():
+    st.markdown("<h2 style='text-align: center'>CAA Violators</h2>",
+                unsafe_allow_html=True)
+    # this is where graph for CAA Violators
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+    st.markdown("<h2 style='text-align: center'>CWA Violators</h2>",
+                unsafe_allow_html=True)
+    # this is where graph for CWA Violators
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+    st.markdown("<h2 style='text-align: center'>RCRA Violators</h>",
+                unsafe_allow_html=True)
+    # this is where graph for RCRA Violators
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+
+# Bottom info for the tabs
+
+
+def bottom_info():
+    st.header("")  # creates a gap
+
+    # Define a CSS style for the gray rectangle with curved edges
+    gray_rectangle_style = """
+    background-color: #e6e6e6;
+    padding: 10px;
+    border-radius: 5px;
+    color: black;
+    font-size: 12px;
+  """
+
+    # text for the bottom of the page
+    text = """The reliability of data in figures throughout this report is 
+    indicated by the figure subtitle and degree of transparency. See the Data 
+    Limitations section to view the transparency-coding table. Access state and 
+    congressional district data here."""
+
+    # Use st.markdown to display the gray rectangle with text
+    st.markdown(
+        f'<div style="{gray_rectangle_style}">{text}</div>', unsafe_allow_html=True)
+    st.header("")  # creates a gap
+
+# data visualization for the sidebar
```

Comment:
Is this comment referring to fillSideBar? If so it will be nice to have it right above it :)